products/opensuse/main.pm

```diff
@@ -417,8 +417,7 @@ elsif (get_var('DOCKER_IMAGE_TEST')) {
     loadtest 'console/docker_image';
 }
 elsif (get_var('PUBLIC_CLOUD')) {
-    boot_hdd_image;
-    loadtest "publiccloud/ipa";
+    load_publiccloud_tests;
```

Comment:
Would it be possible for you to move this even into load_common_opensuse_sle_tests? Then there would be no duplication left

Reply:
I tried, but then there are more test modules loaded. Should that be so?